avm/res/synapse/workspace/main.bicep

```
@@ -322,6 +325,19 @@ resource workspace_roleAssignments 'Microsoft.Authorization/roleAssignments@2022
   }
 ]
 
+// Firewall Rules
+module workspace_firewall_rules 'firewall-rules/main.bicep' = [
```

```suggestion
module workspace_firewallRules 'firewall-rules/main.bicep' = [
```
Please change to camelCase.